tests/test_journey_methods_comprehensive.py

```
@@ -0,0 +1,468 @@
+    assert "transition_ticket" in result["message"]
+    mock_client.work_items.comments.create.assert_called_once()
+    mock_client.work_items.update.assert_not_called()
+
+
+@patch('plane_mcp.journey.tools.read.get_plane_client_context')
+def test_search_tickets_unknown_label_returns_warning(mock_get_context, resolver, mock_client):
+    mock_get_context.return_value = (mock_client, "test-workspace")
+    journey = ReadJourney(resolver)
+
+    # Only 'bug' exists; 'phantom' does not
+    existing_label = MagicMock(); existing_label.name = "bug"; existing_label.id = "lbl-1"
```

Ruff (0.15.9)

[error] Multiple statements on one line (semicolon)

(E702)